iphone/Maps/UI/Settings/MWMTTSSettingsViewController.mm

```diff
@@ -41,6 +42,9 @@ virtual void SelectCell(UITableView * /* tableView */, NSIndexPath * /* indexPat
   virtual ~BaseCellStategy() {}
 };
 
+UITableViewCell* voiceInstructionsCell;
+UITableViewCell* streetNamesCell;
```

Comment:
At least it will hide the variable visibility outside of the mm file.
```suggestion
static UITableViewCell* voiceInstructionsCell;
static UITableViewCell* streetNamesCell;
```